app/views/admin/world_location_news/show.html.erb

```diff
@@ -1,24 +1,31 @@
-<% page_title @world_location_news.name %>
-<% page_class "world-location-show" %>
+<% content_for :page_title, @world_location_news.name %>
+<% content_for :title, @world_location_news.name %>
+<% content_for :title_margin_bottom, 4 %>
 
-<%= content_tag_for(:div, @world_location_news) do %>
-  <div class="world-location-header">
-    <h1>
-      <span class="name"><%= @world_location_news.name %></span>
-    </h1>
-    <p>
-    <%= link_to "View on website", @world_location_news.public_url(cachebust_url_options) %>
-  </div>
+<%= link_to "View on website", @world_location_news.public_url(cachebust_url_options), class: "govuk-link" %>
 
-  <section class="world-location-details">
-    <%= tab_navigation_for(@world_location_news) %>
+<%= render "components/secondary_navigation", {
+  aria_label: "Document tabs",
+  items: [
+    {
+      label: "Details",
+      href: admin_world_location_news_path(@world_location_news),
+      current: true,
+    },
+    {
+      label: "Translations",
+      href: admin_world_location_news_translations_path(@world_location_news),
+    },
+    {
+      label: "Features (#{I18n.locale})",
+      href: features_admin_world_location_news_path(@world_location_news, locale: I18n.locale),
```

Comment:
So i think the behaviour of this is subtly different to the current implementation

```
  def world_location_news_tabs(world_location)
    tabs = {
      "Details" => admin_world_location_news_path(world_location),
      "Translations" => admin_world_location_news_translations_path(world_location),
      "Features (#{Locale.new(:en).native_language_name})" => features_admin_world_location_news_path(world_location, locale: I18n.default_locale),
    }
    world_location.non_english_translated_locales.each do |locale|
      tabs["Features (#{locale.native_language_name})"] = features_admin_world_location_news_path(world_location, locale: locale.code)
    end
    tabs
  end
```

This ☝️ creates a features tab for the English translation and then an additional features  tab for each non-english translation.

You could do something like

```
<%= render "components/secondary_navigation", {
  aria_label: "Document tabs",
  items: [
    {
      label: "Details",
      href: admin_world_location_news_path(@world_location_news),
      current: true,
    },
    {
      label: "Translations",
      href: admin_world_location_news_translations_path(@world_location_news),
    },
    {
      label: "Features (#{Locale.new(:en).native_language_name})",
      href: features_admin_world_location_news_path(@world_location_news, locale: I18n.locale),
    },
    world_location.non_english_translated_locales.map do |locale|
      {
        label: "Features (#{locale.native_language_name})",
        href: features_admin_world_location_news_path(world_location, locale: locale.code),
      },
    end
  ].compact
}%>

```